README.md

```diff
@@ -10,29 +10,94 @@ For general information about developing packages, see the Dart guide for
 and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages). 
 -->
+# miquido_flutter_lints
+##### This is collection of Flutter lints that we use and follow in [Miquido Software development company](https://www.miquido.com).
 
-This is collection of Flutter lints that we share and use in Miquido Sofware House. This is strict, large and rouboust collection beacuse we strive for high quality code and we want follow the same rules in all of our mobile apps.
+This is strict, large and robust collection beacuse we strive for high quality code, good practices and we want follow the same coding style in all our mobile apps.
+
+And we just looooove lots of lints.
 
 ## Features
 
```

Comment:
I would suggest removing `Features` section completely :P 
Lints are pretty self-explanatory and I can see that others https://pub.dev/packages/very_good_analysis and https://pub.dev/packages/flutter_lints also don't have it :P

Reply:
removed